.github/workflows/publish.yml

```diff
@@ -0,0 +1,40 @@
+name: publish
+
+on:
+  workflow_call:
+    inputs: 
+      source:
+        required: true
+        type: string
+    secrets:
+      apikey:
+        required: true
```

Comment:
This one for build/test/publish to ...somewhere...

It takes a secret as a parameter. This may not be necesary if it can access organization secrets, but I didn't fork this to my github org, so i couldn't test that.

It also takes a `NuGet` source. This was necesary for Cloudsmith, because the two were setup as different Cloudsmith repos, which gives them a different NuGet endpoint. 

Not sure what the longer term intention is with CloudSmith. But it would probably make more sense for them to all be one CloudSmith repository, to get one endpoint, and then we can either drop this param, or pass a boolean, of NuGet or Cloudsmith. 
Just to keep from having to update the child repositories.

Reply:
Yeah, I had no idea either ;)

I'll keep it as a parameter for now, and move to the other Cloudsmith repo.

Reply:
Sounds good.

Reply:
Moved to use a default value (of cloudsmith) for the moment.

Secrets still need to be passed